README.md

```diff
@@ -0,0 +1,108 @@
+# Expresso Weather
+
+Expresso Weather is a JSON API that uses the Dark Sky API along with the Google Geocoding API to return weather information based on a location provided by the user. The user must include a valid API key. The data returned includes houlry and daily objects base on the current weather. These objects include a summary, icon, temperature, percipitation percentages, as well as wind and visibility information. 
+
+## Learning Goals
+  - Utilize a project board to create and track details for project completion.
+  - Practice written technical communication with concise and consistent git commits and clear pull requests.
+  - Implement testing in JavaScript.
```

Comment:
Implement testing with Jest? Maybe add a reference to Jest.

Reply:
Thank you, I will make it a link.